src/DisplayPreference.php

```diff
@@ -466,6 +467,38 @@ class='btn'>";
         echo "</div>";
     }
 
+
+    /**
+     * Return the group name of an element in the searchopt array
+     *
+     * The group names are located before the items that belong to it, and are the only string keys, every item's key are integer.
+     *
+     * We first get the keys of the array to be able to iterate trought his items, including the group names.
+     * So we iterate trought the array key's in a reverse order,
+     * starting from the position before the item which we want to get the group name.
+     * The first key of string type we encouter, is our item's group name.
+     *
+     * @param array $searchopt
+     * @param int   $searchoptkey
+     *
+     * @return string Return the name of the group or an empty string.
+     *
+     * @since 10.0.8
+     */
+    private static function nameOfGroupForItemInSearchopt($searchopt, $searchoptkey)
```

Comment:
1. `static` methods should avoided when it is possible.
2. Parameters types and return values types should be defined when it does not introduce a breaking change.
```suggestion
    private function nameOfGroupForItemInSearchopt(array $search_options, int $search_option_key): string
```